pkg/didcomm/crypto/jwe/authcrypt/encrypt.go

```diff
@@ -18,24 +18,29 @@ import (
 	chacha "golang.org/x/crypto/chacha20poly1305"
 	"golang.org/x/crypto/poly1305"
 
-	jwecrypto "github.com/hyperledger/aries-framework-go/pkg/didcomm/crypto"
+	"github.com/hyperledger/aries-framework-go/pkg/wallet"
 )
 
 // Encrypt will JWE encode the payload argument for the sender and recipients
 // Using (X)Chacha20 encryption algorithm and Poly1305 authenticator
 // It will encrypt using the sender's keypair and the list of recipients arguments
-func (c *Crypter) Encrypt(payload []byte, sender jwecrypto.KeyPair, recipients [][]byte) ([]byte, error) { //nolint:lll,funlen
-	err := verifyKeys(sender, recipients)
-	if err != nil {
-		return nil, fmt.Errorf("failed to encrypt message: %w", err)
+func (c *BaseCrypter) Encrypt(payload, senderKey []byte, recipients [][]byte) ([]byte, error) { //nolint:funlen
+	if len(senderKey) == 0 {
+		return nil, fmt.Errorf("failed to encrypt message")
```

Comment:
error message should include something like empty sender key

Reply:
will update...

Reply:
done